src/main/java/org/apache/sysml/runtime/controlprogram/paramserv/ParamservUtils.java

```diff
@@ -131,7 +133,22 @@ public static MatrixBlock generatePermutation(int numEntries) {
 		return permutation;
 	}
 
-	public static ExecutionContext createExecutionContext(ExecutionContext ec, String updFunc, String aggFunc, int workerNum, int k) {
+	public static String[] getCompleteFuncName(String funcName, String prefix) {
+		String[] keys = DMLProgram.splitFunctionKey(funcName);
+		String namespace = null;
+		String func = keys[0];
+		if (keys.length == 2) {
+			namespace = keys[0];
+			func = keys[1];
+		}
```

Comment:
You can simplify that because we're only dealing with out own function key - so there should be no need to handle empty namespaces.
```
return StringUtils.isEmpty(prefix) ? keys :
	new String[] { keys[0], prefix + keys[1] };
```

Reply:
Well, in my negative test, I used some dummy update function which is in the empty namespace. Hence, I'd like to keep it in order to handle this case.